### PR TITLE
Symbol error

### PR DESCRIPTION
**Pull request by tim-zju**
symbol errors in quotation mark( change “” to "" , ‘’ to '')

### CODE REVIEW

File: _includes/partner-script.js [hunk 1]
```diff
@@ -54,7 +54,7 @@
 			name: 'Skippbox',
 			logo: 'skippbox',
 			link: 'http://www.skippbox.com/tag/products/',
-			blurb: 'Creator of Cabin the first mobile application for Kubernetes, and kompose. Skippbox’s solutions distill all the power of k8s in simple easy to use interfaces.'
+			blurb: 'Creator of Cabin the first mobile application for Kubernetes, and kompose. Skippbox's solutions distill all the power of k8s in simple easy to use interfaces.'
 		},
 		{
 			type: 0,
```
Comment (xulike666) [inline _includes/partner-script.js:60]: What's this and please revert it.

File: docs/getting-started-guides/meanstack.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ $ ls
 Dockerfile app
 ```
 
-Let’s build.
+Le's build.
```
Comment (xulike666) [inline docs/getting-started-guides/meanstack.md]: please revert this.

**Review by xulike666 — CHANGES_REQUESTED** [review]
please make sure all of changes are correct.

File: docs/getting-started-guides/meanstack.md (hunk 2 above)
Comment (tim-zju) [reply docs/getting-started-guides/meanstack.md]: ok, i have revert this opertion

File: _includes/partner-script.js (hunk 1 above)
Comment (tim-zju) [reply _includes/partner-script.js:60]: ok, i have revert this opertion

File: docs/user-guide/managing-deployments.md [hunk 3]
```diff
@@ -85,8 +85,8 @@ NAME           CLUSTER-IP   EXTERNAL-IP   PORT(S)      AGE
 my-nginx-svc   10.0.0.208                 80/TCP       0s
 ```
 
-With the above commands, we first create resources under docs/user-guide/nginx/ and print the resources created with `-o name` output format 
-(print each resource as resource/name). Then we `grep` only the "service", and then print it with `kubectl get`. 
+With the above commands, we first create resources under docs/user-guide/nginx/ and print the resources created with `-o name` output format
+(print each resource as resource/name). Then we `grep` only the "service", and then print it with `kubectl get`.
```
Comment (xulike666) [inline docs/user-guide/managing-deployments.md]: Please do not change the default markdown format such as adding an unnoticed short blank at the end of line.

**Review by xulike666 — CHANGES_REQUESTED** [review]
Please make sure that do not change the default markdown format.

File: docs/user-guide/managing-deployments.md (hunk 3 above)
Comment (tim-zju) [reply docs/user-guide/managing-deployments.md]: i fixed these problems

**Review by xulike666 — APPROVED** [review]

**Comment by xulike666** [comment]
LGTM

**Pull request merged**